src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,155 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private StorageDao storageDao;
+    private User actualuser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        actualuser = new User();
+        actualuser.setLogin("sidorov");
+        actualuser.setPassword("123456");
+        actualuser.setAge(35);
+    }
+
+    @Test
+    void age_18_Ok() {
+        actualuser.setAge(18);
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
```

Comment:
The test method name `age_18_Ok` does not follow the naming convention described in the checklist. It should be named `register_age18_Ok` to reflect the method under test, the state, and the expected behavior.